test/llvmpasses/Makefile

```diff
@@ -0,0 +1,11 @@
+SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+JULIAHOME := $(abspath $(SRCDIR)/../..)
+include $(JULIAHOME)/deps/Versions.make
+include $(JULIAHOME)/Make.inc
+include $(JULIAHOME)/deps/llvm-ver.make
+BUILDDIR = $(JULIAHOME)/deps/scratch
+include $(JULIAHOME)/deps/llvm-options.mk
+LLVM_SRC_DIR:=$(JULIAHOME)/deps/srccache/llvm-$(LLVM_VER)
```

Comment:
These configuration options aren't really right, since they assume a very specific in-tree source build layout. Does LLVM not provide any way to run its test framework out-of-tree?